site/themes/s2b_hugo_theme/static/css/custom.css

```diff
@@ -71,6 +71,22 @@ main .donate a {
   }
 }
 
+.customers .item {
+  margin-top: 2px;
```

Comment:
Would be great to move to rem values

Reply:
For this to work and reveal the outline completely, the size needs to be 4px (2px offset and 2px outline width)